src/Core/src/Platform/iOS/KeyboardAutoManagerScroll.cs

```diff
@@ -257,7 +266,26 @@ internal static async Task AdjustPositionDebounce()
 
 		var entranceCount = DebounceCount;
 
-		await Task.Delay(10);
```

Comment:
This delay is not really neccessary now. This was added in originally because clicking the entry triggers an event and the keyboard rising triggers an event. These events were sort of racing and we needed information about the cursor coming from the entry trigger. Now, the cursor logic is moved to later in the process and now we don't really need this delay. I would consider leaving this in just in case, but there is a small animation hiccup with this delay, and without it, the scrolling is MUCH more smooth!